ipv6/kustomization.yaml

```diff
@@ -1,9 +1,10 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 configMapGenerator:
-- behavior: replace
+- behavior: merge
   literals:
   - PROVISIONING_IP=fd2e:6f44:5dd8:b856::2
+  - DHCP_RANGE=fd2e:6f44:5dd8:b856::3,fd2e:6f44:5dd8:b856::ff
```

Comment:
In the v4 sample we have a range of `.10` to `.100` - we should perhaps do the same here (particularly the start range to avoid potential conflict with the cluster-hosted ironic in the openshift case as that uses `.3` atm?)

Reply:
Sure, so `::A` to just `::64`?

Reply:
yup, just so we know there's no chance of the DHCP leases overlapping with other static addresses at the start of the subnet range.

Reply:
I suppose we could leave the upper bound at the subnet end, and just ensure the start is offset from the beginning?

Reply:
I like the idea of giving it a default that will be big enough for production and avoids the start of the range, something like a /96
DHCP_RANGE=fd2e:6f44:5dd8:b856:0:1:0:0,fd2e:6f44:5dd8:b856:0:1:fff:fff